docs/src/corpus.md

```diff
@@ -4,35 +4,54 @@ Working with isolated documents gets boring quickly. We typically want to
 work with a collection of documents. We represent collections of documents
 using the Corpus type:
 
-    crps = Corpus([StringDocument("Document 1"),
-                   StringDocument("Document 2")])
+```julia
+julia> crps = Corpus([StringDocument("Document 1"),
+                      StringDocument("Document 2")])
+Corpus{StringDocument{String}}(StringDocument{String}[StringDocument{String}("Document 1", DocumentMetadata(English(), "Unnamed Document", "Unknown Author", "Unknown Time")), StringDocument{String}("Document 2", DocumentMetadata(English(), "Unnamed Document", "Unknown Author", "Unknown Time"))], 0, Dict{String,Int64}(), Dict{String,Array{Int64,1}}(), TextHashFunction(hash, 100))
+```
 
 ## Standardizing a Corpus
 
-A `Corpus` may contain many different types of documents:
+!!! note
```

Comment:
remove this. people read what they want, let's not try and force them to do something. this does not add anything to the docs.

Reply:
I am marking this resolved via the latest pushed commit.